src/script/vco-ie/main_warhammer/wh3_main_ksl_the_great_orthodoxy/missions.lua

```diff
@@ -0,0 +1,185 @@
+local missions = {
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_ksl_ort_1_there_can_only_be_one;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+
+				objective
+				{
+					override_text mission_text_text_vco_ksl_ort_1_there_can_only_be_one;

```

Comment:
I think you can merge this one and the `CONTROL_N_REGIONS_INCLUDING` one. Use vco_dummy only when there is no alternative.